test/hexagon/codegen/valign.cpp

```diff
@@ -0,0 +1,39 @@
+// RUN: ./valign.out | FileCheck %s
+#include <Halide.h>
+#include "halide-hexagon-setup.h"
+#include <stdio.h>
+using namespace Halide;
+using namespace Halide::Internal;
+IRPrinter irp(std::cerr);
+//CHECK: = vmem(
+//CHECK: valign(v{{[0-9]+}},v{{[0-9]+}},#4)
+//CHECK-NOT: = vmemu
+
+void check_valign(Target target) {
+  Halide::Var x("x"), y("y");
```

Comment:
Yes, this is a todo. We had been using 2-space indentation internally and that crept into our Halide code. The understanding between Dillon and me is that we keep it at 2-space until the hexagon branch is ready to merge into master. A clean sweep to change indentation would be performed then.

So, for now, my rule is that if I issue a pull-request to master, I'll make sure the indentation is 4-space. If on hexagon, then we'll wait until later to fix it. I can however do it now as well. Not a problem. Especially for new files it makes complete sense. Although, this lit based test infrastructure that we have should be deprecated soon as well.